docs/markdown/chapter3/chapter3.md

```diff
@@ -15,7 +15,7 @@ In this chapter you will:
 This section covers the installation and configuration of the Database and search functionality on a Linux server. We will install the ‘ELK’ Stack from Elasticsearch for this portion.
 
 What is the ELK Stack?
-"ELK" is the acronym for three free and open projects: Elasticsearch, Logstash, and Kibana. Elasticsearch is a search and analytics engine. Logstash is a server‑side data processing pipeline that ingests data from multiple sources simultaneously, transforms it, and then sends it to a "stash" like Elasticsearch. Kibana lets users visualize data with charts and graphs in Elasticsearch.
+"ELK" is the acronym for three no-cost to users and open projects: Elasticsearch, Logstash, and Kibana. Elasticsearch is a search and analytics engine. Logstash is a server‑side data processing pipeline that ingests data from multiple sources simultaneously, transforms it, and then sends it to a "stash" like Elasticsearch. Kibana lets users visualize data with charts and graphs in Elasticsearch.
```

Comment:
"ELK" is the acronym for three open projects which come at no cost to users: Elasticsearch, Logstash, and Kibana" or similar, whatever you think best.